client/state/reader/site-blocks/selectors/is-fetching-site-blocks.js

```diff
@@ -13,7 +13,7 @@ import 'state/reader/init';
  * @returns {boolean} true if we are fetching site blocks
  */
 export default function isFetchingSiteBlocks( state ) {
-	const inflightPages = get( state, [ 'reader', 'siteBlocks', 'inflightPages' ] );
+	const inflightPages = state?.reader?.siteBlocks?.inflightPages;
```

Comment:
Assuming that `reader` reducer has been loaded, `siteBlocks` and its `inflightPages` will always exist.  Also, what is the chance that we'll call the selector with a `null` or `undefined` `state`?